Documentation/contributing/development/images.rst

```diff
@@ -238,11 +238,15 @@ different value and then proceed with the steps below.
    generate one commit and push it to your branch with all the necessary changes
    across files in the repository. Once this is done the CI can be executed.
 
-#. Update the versions of the images that are pulled into the CI VMs.
+#. After merging the PR, do the following steps to update the versions of the
+   images that are pulled into the CI VMs.
 
 * Open a PR against the :ref:`packer_ci` with an update to said image versions. Once your PR is merged, a new version of the VM will be ready for consumption in the CI.
-* Update the ``SERVER_VERSION``  field in ``test/Vagrantfile`` to contain the new version, which is the build number from the `Jenkins Job for the VMs <https://jenkins.cilium.io/job/Vagrant-Master-Boxes-Packer-Build/>`_. For example, build 119 from the pipeline would be the value to set for ``SERVER_VERSION``.
-* Open a pull request with this version change in the cilium repository.
+* Update the ``SERVER_VERSION`` field in ``vagrant_box_defaults.rb`` to contain the new version, which is the build number from the `Jenkins Job for the VMs <https://jenkins.cilium.io/job/Vagrant-Master-Boxes-Packer-Build/>`_. For example, build 119 from the pipeline would be the value to set for ``SERVER_VERSION``.
+* After merging the `packer-ci-build`_ PR, open a pull request with this version
+  change in the cilium repository.
+
+.. _packer-ci-build: https://github.com/cilium/packer-ci-build/
```

Comment:
Not sure if adding the link again here is a good idea, we already link to ``:ref:`packer_ci` `` above. Anyway. Doesn't matter much.

Reply:
Yep, where you can find the link to https://github.com/cilium/packer-ci-build/ in the first paragraph. So if the URL of this repo ever changes, that makes one more reference to update. On the other hand, it's one fewer click to find the repo :shrug:. Doesn't matter really, let's keep your version.